depx/graph.py

```diff
@@ -19,10 +21,24 @@ def create_graph_from(dependencies):
     return graph
 
 
+def get_html_template():
+    default = Path('depx') / 'template.html'
```

Comment:
I think is safer use the absolute path: `Path(__file__).parent.absolute()`

Reply:
Indeed : )